severity-report.Rmd

```diff
@@ -122,62 +122,61 @@ We calculated the weekly proportion of positive tests that were S-gene negative
+where $i$ indicates UTLA or NHS region, $w$ the week containing day t, $\epsilon$ is an error term that accounts for imported deaths/admissions not linked to local cases/admissions, $f_{i,w}$ is the fraction of cases that were found to be S-gene negative by UTLA each week, and $\phi$ is the overdispersion of admissions/deaths. The case-fatality ratio (or hospitalisation-fatality / case-hospitalisation ratio, respectively) of S-gene negative cases was then assumed to be a function of local variation, and case/admissions burden reflecting additional mortality that might occur due to hospital pressure,
 
 $$c^{+}_{i,t} = \mathrm{logit}^{-1}\left(\gamma_{i} + \beta C_{i,t}\right)$$
-where $\gamma_{i}$ is an intercept corresponding to the baseline case fatality rate per UTLA or NHS region, and $\beta$ corresponds to the effect of case burden with non-linearity incorporated using a thin plate spline. The S-gene negative case fatality rate was then assumed to be related to the S-gene positive case fatality rate via a multiplicative relationship,
+where $\gamma_{i}$ is an intercept corresponding to the baseline case-fatality ratio per UTLA or NHS region, and $\beta$ corresponds to the effect of case burden with non-linearity incorporated using a thin plate spline. The S-gene negative case-fatality ratio was then assumed to be related to the S-gene positive case-fatality ratio via a multiplicative relationship,
 
 $$c^{-}_{i,t} = \alpha c^{+}_{i,t}$$
 
 or an additive relationship
 
 $$c^{-}_{i,t} = \alpha + c^{+}_{i,t}$$
 
-where $\alpha$ represents either the multiplicative change in case fatality rate or the additive change. These alternative parametrisations represent either a population wide effect for the former parametrisation or a subpopulation effect in the latter parametrisation.
+where $\alpha$ represents either the multiplicative change in case-fatality ratio or the additive change. These alternative parametrisations represent either a population wide effect for the former parametrisation or a subpopulation effect in the latter parametrisation.
 
 ### Convolution analysis
 
-We assumed that the observed number of Covid-19 admissions/deaths ($D_{i,t}$) by date of report were a function of Covid-19 notifications/admissions ($C_{i,t}$) by date of report, convolved by a log normal delay, and scaled by a rate ($c_{i,t}$, which when using cases and deaths is the case fatality rate),
+We assumed that the observed number of Covid-19 admissions/deaths ($D_{i,t}$) by date of report were a function of past Covid-19 notifications/admissions ($C_{i,t-\tau}$, $\tau \in \{0, ..., 30\}$), each convolved by a log-normal delay and scaled by a factor ($c_{i,t}$, which when using cases and deaths is the case-fatality ratio),
 
-$$D_{i,t} \sim \mathrm{NB}\left(c_i\sum_{\tau = 0}^{30} \xi_{i, \tau} C_{i, t-\tau},  \phi \right)$$
+$$D_{i,t} \sim \mathrm{NB}\left(c_i\sum_{\tau = 0}^{30} \xi_{i}(t-\tau | \mu, \sigma) C_{i, t-\tau},  \phi \right)$$
```

Comment:
I'm not sure $\xi$ should be referenced by t as it doesn't evolve over time and I think the interpretation is pmf of being delayed by 2 days * cases 2 days ago?

Reply:
if it did evolve over time I would have thought it would be $\xi_{t, \tau}$

Reply:
oh I see it's the whole convolution rather than the lognormal - I was looking at the rmarkdown rather than the pdf so I think I got lost in the latex...

Reply:
So  just $\xi_{i}(\tau|\mu, \sigma)$

Reply:
(this is where notebooks are nice as you get on the fly rendering of the latex chunks!)